.storybook/customTheme.ts

```diff
@@ -0,0 +1,32 @@
+import { create } from '@storybook/theming/create';
```

Comment:
super small: does this work?
```suggestion
import { create } from '@storybook/theming';
```

Reply:
https://storybook.js.org/docs/7.5/configure/theming#create-a-theme-quickstart

thats how they said to useit